pkg/cloudcore/apis/config/default.go

```diff
@@ -0,0 +1,75 @@
+package config
+
+import (
+	"path"
+
+	"github.com/kubeedge/kubeedge/common/constants"
+	commonconfig "github.com/kubeedge/kubeedge/pkg/common/apis/config"
+)
+
+func NewDefaultCloudCoreConfig() *CloudCoreConfig {
```

Comment:
please add comments on top of all exportable functions

Reply:
ok